openslides/users/static/templates/users/user-form.html

```diff
@@ -44,7 +44,15 @@ <h1 ng-if="!user.id" translate>New participant</h1>
   <div class="form-group row">
     <div class="col-xs-6">
       <label for="inputDefaultPassword" translate>Default password</label>
-      <input type="text" ng-model="user.default_password" class="form-control" name="inputDefaultPassword">
+      <div class="input-group">
```

Comment:
Is it self-explaining the you first have to submit the update form and then come again to this place and click reset?